resources/views/tools/database/edit-add-bread.blade.php

```diff
@@ -153,7 +153,7 @@
 
                     <div class="panel panel-primary panel-bordered">
                         <div class="panel-heading">
-                            <h3 class="panel-title panel-icon"><i class="voyager-window-list"></i> {{ __('voyager.database.edit_rows', ['table' => $table]) }}:</h3>
+                            <h3 class="panel-title panel-icon"><i class="voyager-window-list"></i> {{ __('voyager.database.edit_rows', ['table' => $table]) }}</h3>
```

Comment:
I really don't care much either way, but for my own curiosity, any particular reason for removing the colon?

Reply:
Hello, in the translation in Spanish it is duplicated.
But checking the other translations only happens, it would be better to change it only in the translation.
<img width="373" alt="screen shot 2017-11-08 at 15 24 10" src="https://user-images.githubusercontent.com/12446271/32575172-e730c05e-c498-11e7-8aa8-0ff54bae193b.png">

Reply:
Agreed, I'd remove it from the translation(s) and add it back on this line.

Reply:
Change done!